tests/test_includeme.py

```diff
@@ -37,7 +37,7 @@ def test_present_in_heartbeat(self):
 
     def test_returns_false_if_connection_fails(self):
         with mock.patch.object(self.app.app.registry.indexer, "client") as client:
-            client.is_alive.side_effect = AlgoliaException
+            client._transporter.read.side_effect = AlgoliaException
```

Comment:
This `side_effect` property doesn't exist. Is this test passing?

Reply:
It is because client has been mocked on line 39 here.